src/testRunner/unittests/tscWatch/watchEnvironment.ts

```diff
@@ -725,6 +725,38 @@ describe("unittests:: tsc-watch:: watchEnvironment:: tsc-watch with different po
         verify(/*fsWatchWithTimestamp*/ false);
     });
 
+    describe("with fsWatch with fsWatchWithTimestamp with useFsEventsOnParentDirectory", () => {

```

Comment:
It's actually quite tricky to make the regression test fail on main. In the test environment, the callback is receiving a defined currentModifiedTime, but in an actual project it receives an undefined currentModifiedTime.